certified-connectors/WorkSpan/apiDefinition.swagger.json

```diff
@@ -1521,6 +1635,235 @@
+    },
+    "PersonBaseDto": {
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "id"
+      ],
+      "type": "object"
+    },
+    "ReportBaseDto": {
+      "properties": {
+        "createdAt": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "createdBy": {
+          "$ref": "#/definitions/PersonBaseDto"
+        },
+        "description": {
+          "type": "string"
+        },
+        "id": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "modifiedAt": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "modifiedBy": {
+          "$ref": "#/definitions/PersonBaseDto"
+        },
+        "name": {
+          "type": "string"
+        },
+        "reportStatus": {
+          "enum": [
+            "DRAFT",
+            "PUBLISHED"
+          ],
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "name"
+      ],
+      "type": "object"
+    },
+    "ReportPageDto": {
+      "properties": {
+        "number": {
+          "format": "int64",
+          "minimum": 0,
+          "type": "integer",
+          "default": 0
+        },
+        "size": {
+          "format": "int64",
+          "maximum": 10000,
+          "minimum": 1,
+          "type": "integer",
+          "default": 500
+        }
+      },
+      "required": [
+        "number",
+        "size"
+      ],
+      "type": "object"
+    },
+    "ReportDataRequest": {
+      "properties": {
+        "filters": {
+          "$ref": "#/definitions/ReportFilterContainer"
+        },
+        "page": {
+          "$ref": "#/definitions/ReportPageDto"
+        }
+      },
+      "type": "object"
+    },
+    "ReportDataResponse": {
+      "properties": {
+        "endOfList": {
+          "type": "boolean"
+        },
+        "results": {
+          "items": {
+            "additionalProperties": true
+          },
+          "type": "array"
+        },
+        "version": {
+          "format": "int64",
+          "type": "integer"
+        }
+      },
+      "type": "object"
+    },
+    "ReportFieldFilter": {
+      "properties": {
+        "colId": {
+          "type": "string"
+        },
+        "filter": {
+          "$ref": "#/definitions/ReportFilter"
+        },
+        "type": {
+          "enum": [
+            "TEXT",
+            "TEXT_REPEATED",
+            "TEXT_PG_ARRAY",
+            "BOOLEAN",
+            "NUMBER",
+            "DECIMAL",
+            "DATE",
+            "DATE_OFFSET",
+            "DATE_TIME",
+            "DATE_TIME_OFFSET",
+            "CURRENCY"
+          ],
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "ReportFilter": {
+      "properties": {
+        "negate": {
+          "type": "boolean",
+          "title": "",
```

Comment:
@WorkSpan-Engineering Empty property values can be removed until unless you have a specific reason to keep them.